src/methods/query/index.ts

```diff
@@ -147,13 +148,23 @@ async function getPoolInfo() {
     
 }
 
-async function getSpotPrice(asset1: string, asset2: string) {
+async function getSpotPrice(asset1Id: string, asset2Id: string) {
   return new Promise(async (resolve, reject) => {
     try {
       const api = Api.getApi();
     
       if (api) {
-        const amount = new BigNumber(await wasm.get_spot_price(asset1, asset2, '1000000000000'));
+
+        const assetsAmounts = await getAssetsAmounts(asset1Id, asset2Id);
```

Comment:
let's add this to SDK also as getPoolAssetAmounts(asset1Id, asset2Id) => { asset1Amount: BigNumber, asset2Amount: BigNumber }